bin/vhost

```diff
@@ -174,10 +175,33 @@ fi
 #═════════════════════════════════════════════════════════════════════════════════════════════════
 if [ "$command" = "install" ]; then
   require_config_dir
+
   ip=127.0.0.1
-  hosts=$( jq -rM '.hosts | join(" ")' $config_file )
+  # Merge local config if any
+  if test -f "$local_config_file"; then
+    ptask "Using local config"
+    config=$(jq -s '.[0] * .[1]' $config_file $local_config_file)
+  else
+    config=$(cat $config_file)
+  fi
+  hosts=$(echo $config | jq -rM '.hosts | join(" ")')
+
+  if [[  "$(echo $config  | jq 'has("env")')" == "true" ]]; then
+    env_vars=$(echo $config | jq -rM '.env | to_entries | map(.key + @sh "=\(.value)")| join(" ")' || '')
+    envsubst_vars="\$project_dir,\$config_dir,$(echo $config | jq -rM '.env | to_entries | map("$" + .key)| join(",")' || '')"
+  else
+    env_vars=""
+    envsubst_vars="\$project_dir,\$config_dir"
+  fi
   host_line="$ip $hosts"
-  project_name=$(jq -rM '.name' $config_file)
+  project_name=$(echo $config | jq -rM '.name')
+
+  # Use provided env vars from config
+  if [[ ! -z "${env_vars}" ]]; then
+    echo $env_vars
+    eval "export $env_vars"
+  fi
+  # For configs to be able to reference other sibling files
```

Comment:
What is this comment referring to?

Reply:
LOL, good catch. It's a remnant of my attempt to add something like `project_dir`, then I realized you already had it. I'll remove this.